### PR TITLE
Master

### PR DESCRIPTION
**Pull request by nununo**
READ TABLE mo_repo->get_files_local( ) not valid for 702.

To address issue #601

### CODE REVIEW

**Comment by larshp** [comment]
Thanks 😄

**Comment by EduardoCopat** [comment]
Sorry... 😞

File: src/zabapgit_page_stage.prog.abap [hunk 1]
```diff
@@ -270,9 +270,11 @@ CLASS lcl_gui_page_stage IMPLEMENTATION.
   ENDMETHOD.  "scripts
 
   METHOD read_last_changed_by.
-    DATA: ls_local_file TYPE ty_file_item.
+    DATA: ls_local_file TYPE ty_file_item,
+          t_files_local type ty_files_item_tt.
```
Comment (EduardoCopat) [inline src/zabapgit_page_stage.prog.abap]: could be lt_files_local to follow the conventions

Comment (nununo) [inline src/zabapgit_page_stage.prog.abap]: @EduardoCopat you're right sorry. Variable renamed and new PR #603 submitted.

**Pull request merged**